### ISSUE TITLE
Game launched only once and won't launch again

### ISSUE THREAD

**Issue by Awstein** [marzent/XIV-on-Mac#77]
Attaching the error log:

Unhandled exception: page fault on read access to 0x00000000 in 64-bit code (0x00000003af6ec450).
Register dump:
 rip:00000003af6ec450 rsp:000000000031ee48 rbp:0000000000000000 eflags:00000246 (   - --  I  Z- -P- )
 rax:0000000000000000 rbx:0000000000000000 rcx:0000000000000000 rdx:000000000000002e
 rsi:000000000000011b rdi:000000000031f3d0  r8:0000000000000150  r9:0000000000000030 r10:00000000000007fb
 r11:0000000000000155 r12:000000000031f5b0 r13:000000000031ef80 r14:000000014000a8a0 r15:000000000001b00b
Stack dump:
0x000000000031ee48:  0000000140006524 0000000000000000
0x000000000031ee58:  0000000000000000 0000000000000000
0x000000000031ee68:  0000000000002c00 0000000000000000
0x000000000031ee78:  0000000000020006 0000000000000000
0x000000000031ee88:  000000000031f5b0 0000000000000000
0x000000000031ee98:  0000000000000000 0000000000000000
0x000000000031eea8:  0000000000000000 0000000000000001
0x000000000031eeb8:  0000000000000000 000000007bc2d3d0
0x000000000031eec8:  0000000000000000 000000000031f3a0
0x000000000031eed8:  000000007b635808 000000000031f190
0x000000000031eee8:  000000007bc0dbc0 000000000031ef30
0x000000000031eef8:  0000000000000000 000000ff00000000
Backtrace:
=>0 0x00000003af6ec450 EntryPoint+0x2fffe07e0() in ucrtbase (0x0000000000000000)
  1 0x0000000140006524 in wineboot (+0x6523) (0x0000000000000000)
  2 0x0000000140004445 in wineboot (+0x4444) (0x0000000000000000)
  3 0x000000007b630169 EntryPoint+0xffffffffffffffff() in kernel32 (0x0000000000000000)
  4 0x000000007bc6734c EntryPoint+0xffffffffffffffff() in ntdll (0x0000000000000000)
0x00000003af6ec450 EntryPoint+0x2fffe07e0 in ucrtbase: movzbl	(%rax),%ecx
Modules:
Module	Address					Debug info	Name (9 modules)
PE	        6c010000-        6c016000	Deferred        ws2_32
PE	        7b000000-        7b0f1000	Deferred        kernelbase
PE	        7b600000-        7b81b000	Dwarf           kernel32
PE	        7bc00000-        7bcae000	Dwarf           ntdll
PE	       140000000-       140026000	Dwarf           wineboot
PE	       1c8db0000-       1c8e5f000	Deferred        msvcrt
PE	       32a700000-       32a72d000	Deferred        sechost
PE	       330260000-       3302a3000	Deferred        advapi32
PE	       3af670000-       3af74f000	Dwarf           ucrtbase
Threads:
process  tid      prio (all id:s are in hex)
00000088 wineboot.exe
	0000008c    0
	000000dc    0
000000c4 wineboot.exe
	000000c8    0
	00000114    0
000000d4 winedbg.exe
	000000d8    0
	000000f0    0
000000e0 explorer.exe
	000000e4    0
	000000e8    0
	000000ec    0
000000f4 (D) C:\windows\system32\wineboot.exe
	000000f8    0 <==
	00000130    0
00000104 
	00000108    0
0000010c winedbg.exe
	00000110    0
	00000118    0
	0000011c    0
System information:
    Wine build: crossover-wine-21.1.0-51-g2c943bb9
    Platform: x86_64
    Version: Windows 10
    Host system: Darwin
    Host version: 21.0.1
